README.md

```diff
@@ -4,7 +4,17 @@ A plugin for IntelliJ IDEA that detects outdated comments in Java code
 
 ## Launch instructions
 
-- Inside the `src/main/kotlin/...models/config/ModelFilesConfig.kt` file 
-replace `DATA_DIR` value with your local path to the directory, containing files from [here](https://drive.google.com/drive/folders/1E4XOJHfEWOlHXBPxPu3_bkKfuU4G_N49?usp=sharing)
+### Plugin
+
+- Inside the `comment-updater-plugin/src/main/kotlin/...models/config/ModelFilesConfig.kt` file 
+replace default `dataDir` argument with path to the directory, containing files from [here](https://drive.google.com/drive/folders/1E4XOJHfEWOlHXBPxPu3_bkKfuU4G_N49?usp=sharing)
   
-- Run  `gradle/intellij/runIde` task
\ No newline at end of file
+- Run  `gradle/comment-updater-plugin/intellij/runIde` task
+
+### Headless Plugin 
+
+- run `commentupdater.sh` file with three arguments: 
```

Comment:
please, add deatails about the headless plugin purpose. imagine a user which opens CommentUpdater project on GitHub first time and make their "diving" into the project simpler by adding details to README. why do we need headnless part? what does it do? how it could be useful?